client/src/services/environment.ts

```diff
@@ -60,10 +60,45 @@ function getTrialServers(): Array<string> {
   return DEFAULT_TRIAL_SERVERS;
 }
 
+const CLEAN_APP_VERSION = `${APP_VERSION.replace('+dev', '')}`;
```

Comment:
Note: For nightly build, `+{local}` is set to a short commit id (it replace `dev`).

I'm think If it would be simple to handle that part with `misc/version_updater.py` :thinking:, we already have a function (`hide_patch_version`) that can remove the patch version of python, we could have something similar for `CLEAN_APP_VERSION`

Reply:
I update the cleaning part slightly to remove anything after the `+` (if present). Probably good enough.